core/domain/suggestion_services_test.py

```diff
@@ -6755,6 +6755,39 @@ def test_get_returns_empty_if_suggestions_have_waited_less_than_threshold(
         self.assertEqual(
             len(info_about_suggestions_waiting_too_long_for_review), 0)
 
+    def test_get_new_suggestions_for_reviewer_notifications_past_threshold(
+        self) -> None:
+        max_suggestions = 3
+        threshold_days = 2
+        creation_time = datetime.datetime(2020, 6, 14, 5)
```

Comment:
Add a timezone

Reply:
Similar to that of gae_models_test, I have mocked the timezone as UTC, so I believe that it is timezone aware.

Reply:
Didn't get that. Mocked the timezone as UTC? where?

Do you mean you use this return value for `utcnow`?

Reply:
Here, https://github.com/oppia/oppia/pull/19045/files#diff-d6673e0f979aebf88318b0a0d9929c908872841675590855f6e1603df99368a1R6771, I am returning the timezone as UTC timezone, isn't this suffice ?

Reply:
It's not that `utcnow` will not give you the right utc time value. But when you use that datetime object anywhere, python will use your local time zone for conversions if the object doesn't have one.

So for e.g.
my machine is in singapore time zone.
```python
>>> import datetime
>>> ts1 = 1234
>>> dt1 = datetime.datetime.utcfromtimestamp(ts1)
>>> dt1
datetime.datetime(1970, 1, 1, 0, 20, 34)
>>> dt1.timestamp()
-25766.0
>>> dt2 = datetime.datetime.fromtimestamp(ts1, tz=datetime.timezone.utc)
>>> dt2
datetime.datetime(1970, 1, 1, 0, 20, 34, tzinfo=datetime.timezone.utc)
>>> dt2.timestamp()
1234.0
```

So we avoid `utcnow` or `utctimestamp` and naive datetime objects (without timezone) as much as possible.

Reply:
Ah, thanks for the detailed explanation, appreciate it! But then we'll have a similar error to https://github.com/oppia/oppia/pull/19045#discussion_r1470562888, because we would need to update created_on property with the timezone property, but that would mean a lot of changes to make sure we don't break anything. 
I think we should do this in other PR where we use timezone aware object on created_on property. WDYT ?